test/common/http/conn_manager_impl_test.cc

```diff
@@ -3277,5 +3279,34 @@ TEST(HttpConnectionManagerTracingStatsTest, verifyTracingStats) {
   EXPECT_EQ(1UL, tracing_stats.not_traceable_.value());
 }
 
+TEST_F(HttpConnectionManagerImplTest, NoNewStreamWhenOverloaded) {
+  setup(false, "");
+
+  overload_manager_.overload_state_.setState(
+      Server::OverloadActionNames::get().StopAcceptingRequests,
+      Server::OverloadActionState::Active);
+
+  EXPECT_CALL(*codec_, dispatch(_)).Times(1).WillRepeatedly(Invoke([&](Buffer::Instance&) -> void {
```

Comment:
Nit: I know this is in a bunch of places in this file (including some where I copy+pasted tests), but `.Times(1)` is a nop, `EXPECT_CALL` by default implies this.

Reply:
Fixed. Should I also remove all instances of .Times(1) in this file for cleanup?

Reply:
Up to you, cleanup always welcome :)

Reply:
done